content/id/docs/tasks/manage-daemon/rollback-daemon-set.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Melakukan Rollback pada DaemonSet
+content_type: task
+weight: 20
+min-kubernetes-server-version: 1.7
+---
+
+<!-- overview -->
+
+Laman ini memperlihatkan bagaimana caranya untuk melakukan _rollback_ pada sebuah {{< glossary_tooltip term_id="daemonset" >}}.
+
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+Sebelum lanjut, alangkah baiknya jika kamu telah mengetahui cara
+untuk [melakukan _rolling update_ pada sebuah DaemonSet](/docs/tasks/manage-daemon/update-daemon-set/).
+
+<!-- steps -->
+
+## Melakukan _rollback_ pada DaemonSet
+
+### Langkah 1: Dapatkan nomor revisi DaemonSet yang ingin dikembalikan
+
+Lompati langkah ini jika kamu hanya ingin kembali (_rollback_) ke revisi terakhir.
```

Comment:
I think "lewati" is better than "lompati".
```suggestion
Lewati langkah ini jika kamu hanya ingin kembali (_rollback_) ke revisi terakhir.
```

Reply:
I think "lewati" is ambiguous in Indonesian, it can mean "follow" as well, "lompati" means "skip" IMO

Reply:
Ok!